src/database.c

```diff
@@ -1724,7 +1770,7 @@ void tick_login(void) {
 
         newbie=1;
 
-        sprintf(buf,"0000000000�c17%s�c18, a new player, has entered the game.",login.name);
+        sprintf(buf,"0000000000�c17%s�c18, a new player, has entered the game.",login.name);
```

Comment:
Uh oh. This is a regression I did not realize I introduced until looking at the review UI on github. I think my editor mucked with the encoding of the file and converted the ° symbol to this weird <?>